themes/default/content/docs/get-started/_index.md

```diff
@@ -43,7 +43,16 @@ Select one of the following options to learn how to deploy cloud resources using
                 </a>
```

Comment:
I think we should also update the intro paragraph to align with updated messaging:

> Pulumi is a modern infrastructure as code platform that allows you to use familiar programming languages and tools to build, deploy, and manage cloud infrastructure.

Reply:
Fixed in latest commit.